packages/backend/src/peripherals/database/discovery/NotificationManagerRepository.ts

```diff
@@ -0,0 +1,79 @@
+import { Logger, UnixTime } from '@l2beat/shared'
+import { NotificationManagerRow } from 'knex/types/tables'
+
+import { DiscoveryDiff } from '../../../core/discovery/output/diffDiscovery'
+import { BaseRepository, CheckConvention } from '../shared/BaseRepository'
+import { Database } from '../shared/Database'
+
+export interface NotificationManagerRecord {
+  id: number
+  createdAt: UnixTime
+  updatedAt: UnixTime
+  projectName: string
+  blockNumber: number
+  diff: DiscoveryDiff[]
+}
+
+export class NotificationManagerRepository extends BaseRepository {
```

Comment:
Maybe rename to NotificationRepository or UpdateNotificationRepository?
If you go for UpdateNotificationRepository then NotificationManager could be renamed to UpdateNotifier.